docs/endpoints/endpoints.rst

```diff
@@ -0,0 +1,743 @@
+Globus Compute Endpoints
+************************
+
+A Globus Compute Endpoint is a process launched by the user to serve as a conduit for
+executing functions on a computing resource.  The Compute Endpoint process manages the
+site‑specific interactions for executing functions, leaving users with only a single API
+necessary for running code on their laptop, campus cluster, or even leadership‑class HPC
+machines.
+
+In the mental model of Globus Compute, Endpoints are the remote instance:
+
+- Globus Compute SDK |nbsp| --- |nbsp| i.e., the script a researcher writes to submit
+  functions to ...
+
+- Globus Compute Web Services |nbsp| --- |nbsp| the Globus‑provided cloud‑services that
+  authenticate and ferry functions and tasks
+
+- **Globus Compute Endpoint** |nbsp| --- |nbsp| a site-specific process, typically on a
+  cluster head node, that manages user‑accessible compute resources to run tasks
+  submitted by the SDK
+
+The Compute Endpoint may be installed via PyPI in the usual manner, as well as via
+system packages.  See :doc:`installation` for more information.
+
+
+Quickstart
+==========
+
+For those just looking for the quickstart commands:
+
+.. code-block:: console
+
+   $ python3 -m pipx install globus-compute-endpoint
+
+   $ globus-compute-endpoint configure my_first_endpoint
+
+   $ globus-compute-endpoint start my_first_endpoint
+
+   $ globus-compute-endpoint stop my_first_endpoint
+
+
+Getting Started
+===============
+
+Creating New Compute Endpoints
+------------------------------
+
+Create a new endpoint directory and default files in ``$HOME/.globus_compute/`` via the
+``configure`` subcommand:
+
+.. code-block:: console
+
+   $ globus-compute-endpoint configure my_first_endpoint
+   Created profile for endpoint named <my_first_endpoint>
+
+       Configuration file: /.../.globus_compute/my_first_endpoint/config.yaml
+
+The Compute Endpoint will also be in the output of the ``list`` subcommand:
+
+.. code-block:: console
+
+   $ globus-compute-endpoint list
+   +--------------------------------------+--------------+--------------------+
+   |             Endpoint ID              |    Status    |   Endpoint Name    |
+   +======================================+==============+====================+
+   | None                                 | Initialized  | my_first_endpoint  |
+   +--------------------------------------+--------------+--------------------+
+
+The Compute endpoint will receive an endpoint identifier from the Globus Compute web
+service upon first connect.  Until then, it exists solely as a subdirectory of
+``$HOME/.globus_compute/``.
+
+.. _cea_configuration:
+
+As Globus Compute endpoints may be run on a diverse set of computational resources
+(i.e., the gamut from laptops to supercomputers), it is important to configure each
+instance to match the underlying capabilities and restrictions of the resource.  The
+default configuration is functional |nbsp| --- |nbsp| it will process tasks |nbsp| ---
```

Comment:
```suggestion
default configuration below is functional |nbsp| --- |nbsp| it will process tasks |nbsp| ---
```

Reply:
I think "below" in this context would slightly open the room for "is one default different than another default?"  In my mind, there is only one default configuration, and I want to impart that here.  _The_ default configuration ...